scenarios/tracking/01_training_introduction.ipynb

```diff
@@ -127,6 +127,7 @@
     "\n",
```

Comment:
<p>To add to my previous comments: by default the 01 notebook evaluation should not write to disk, or if at all needed then do so without the user noticing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
re-write evaluate().